vale/main.py

```diff
@@ -145,4 +146,6 @@ def main():
     """Download vale if not downloaded and executes it."""
     vale_bin_path = download_vale_if_missing()
 
-    os.execvp(f"{vale_bin_path}", ["vale"] + sys.argv[1:])
+    proc = subprocess.Popen([f"{vale_bin_path}"] + sys.argv[1:])
+    ret_code = proc.wait()
+    sys.exit(ret_code)
```

Comment:
Yes definitely, if we are just waiting for the process, then call is better. 

Regarding future proofing, the docs sort of hint that it's preferred to use subprocess.run, but it's never been clear to me why you'd use it for this case over call (and call is not deprecated).

Reply:
Thanks for that info! I didn't know about that. Let's use `run()` instead.

I'm sorry to ask you all these little changes. I do it just so that you keep the authorship of this contribution :innocent:

Reply:
No problem at all, I appreciate it. I should have mentioned about run/call earlier too.